plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/file/FileMetastoreTableOperations.java

```diff
@@ -63,7 +63,7 @@ protected void commitToExistingTable(TableMetadata base, TableMetadata metadata)
                     currentMetadataLocation, metadataLocation, getSchemaTableName());
         }
 
-        String newMetadataLocation = writeNewMetadata(metadata, version + 1);
+        String newMetadataLocation = writeNewMetadata(metadata, version.orElse(-1) + 1);
```

Comment:
I think it is safe here to do a `version.orElseThrow(...)`. If we are committing to an existing table the version should exist

Reply:
Thanks! Addressed!